### ISSUE TITLE
Unable to install Vanilla OS 2: "failed to name partition: E"

### ISSUE THREAD

**Issue by Yoarhim** [Vanilla-OS/iso-builder#369]
### Issue Description

Every time that I try to install Vanilla OS, I get the same cryptic error: "failed to name partition: E".
I use a unowhy Y13 from 2019, so there is an ssd inside.

### Steps to Reproduce

- Tried to install Vanilla OS
- But it failed

### On what version of Vanilla OS this happens?

Vanilla OS 2 Orchid

### Additional Information

There is the log of the installation process, if it can help:

`
Discarding device blocks: done
Creating filesystem with 262144 4k blocks and 65536 inodes
Filesystem UUID: ad62llef-75ca-47al-9ca4-3fa4f14aa815
Superblock backups stored on blocks:
      32768, 98304, 163840, 229376

Allocating group tables: done
Writing inode tables: done
Creating journal (8192 blocks): done
Writing superblocks and filesystem accounting information: done

panic: failed to run setup operation mkpart: failed to name partition: E
rror: loop disk labels do not support partition name.

goroutine 1 [running]:
main.main()
      github.com/vanilla-os/albius/albius.go:22 +0x106
`

**Comment by dr-msr** [comment]
Similar error happened to me.

Device : ASUS Chrombook C214 custom firmware flashed.

**Comment by mpanhans** [comment]
I am seeing the same error message when trying to install Vanilla OS 2 on a Chuwi tablet.

**Comment by taukakao** [comment]
@mpanhans 
I'm pretty sure this is caused by trying to install to EMMC storage, which is not supported as of now.

If your drive shows up as "mmcblk*" when running "lsblk" in a terminal, then this is the case.

**Comment by mpanhans** [comment]
Yes that sounds right. It is eMMC storage. I do see mmcblk* from lsblk.

**Comment by taukakao** [comment]
If anyone is experiencing this bug and you're not installing to EMMC storage, try going into gparted through the installer and create a gpt partition table on your drive. NOTE: This will delete everything on your drive

Please report back if that was successful.
